tasks/lib/helpers/git.js

```diff
@@ -52,7 +52,7 @@ function checkout(grunt, dir, branchName, done) {
 
 function merge(grunt, dir, mergeFromBranch, msgPrefix, msg, done) {
 
-  exec('git merge --no-ff -m "' + msgPrefix + ' ' + msg + '" ' + mergeFromBranch, {
+  exec('git merge --no-ff --strategy-option theirs -m "' + msgPrefix + ' ' + msg + '" ' + mergeFromBranch, {
```

Comment:
Yes, it's used to merge the temp branch (hotfix or release) into master + develop and `theirs` refers to the temp branch. 

At first I though nothing of it: people should not go play into their `master` branch while in the process of doing a hotfix (same for develop while doing a release). In general, I think I would prefer the process to fail instead of blindly accepting (so an option would be better) but I'd like to first understand why couldn't you merge your hotfix automatically?